test/controllers/foreman/hosts_controller_test.rb

```diff
@@ -58,17 +80,29 @@ def setup
 
     def test_csv_export
       get :content_hosts, params: { :format => 'csv', :organization_id => @host.organization_id }
+
       assert_equal "text/csv; charset=utf-8", response.headers["Content-Type"]
       assert_equal "no-cache", response.headers["Cache-Control"]
-      assert_equal "attachment; filename=\"hosts-#{Date.today}.csv\"",
-        response.headers["Content-Disposition"]
-      buf = response.stream.instance_variable_get(:@buf)
-      assert buf.is_a? Enumerator
-      assert_equal "Name,Subscription Status,Installable Updates - Security,Installable Updates - Bug \
-Fixes,Installable Updates - Enhancements,Installable Updates - Package Count,OS,Environment,\
-Content View,Registered,Last Checkin\n",
-        buf.next
-      assert_equal 3, buf.count
+      assert_equal "attachment; filename=\"hosts-#{Date.today}.csv\"", response.headers["Content-Disposition"]
```

Comment:
this test started failing when i was messing with the other tests. i figured it could use some tidying up!